symengine/tests/eval/test_eval_mpfr.cpp

```diff
@@ -197,6 +198,8 @@ TEST_CASE("precision: eval_mpfr", "[eval_mpfr]")
         std::make_tuple(gamma(add(arg1, arg2)), -100, 100),
         std::make_tuple(gamma(arg1), -100, 100),
         std::make_tuple(gamma(add(add(arg1, arg2), arg1)), -100, 100),
+        std::make_tuple(uppergamma(arg1, arg2), 0.97350097883925,
+                        0.97350097883926),
```

Comment:
I'm not sure why tests pass when this should fail with MPFR<4.

Reply:
I think we should investigate that ---- perhaps the test is not run properly.

Reply:
Looks like `uppergamma(9/4, 2)` is automatically simplified to `(5/4)*exp(-1/4)` and therefore the `uppergamma` eval is not used.